### ISSUE TITLE
Error occurred: -4. Function not yet implemented

### ISSUE THREAD

**Issue by DPB61** [0x1mason/GribApi.NET#81]
I am copying one of the examples I found in the Issues area, using beta 4 and converted to vb.net, but seems properties like GeoSpatialValues have changed to GridCoordinateValues, etc so I think I have fixed them.

However, when I run this, I get an error on the 3rd line:
"Grib.Api.Interop.GribApiException occurred
  HResult=0x80131500
  Message=Error occurred: -4. Function not yet implemented"

```
Using gribFile As New GribFile(txtGRIB2File.Text)
            For Each hour As GribMessage In gribFile
                For Each geoLocValue As GridCoordinateValue In hour.GridCoordinateValues
                    If geoLocValue.IsMissing Then
                        Continue For
                    End If
                    Dim lat As Double = geoLocValue.Latitude
                    Dim lon As Double = 180.0 - geoLocValue.Longitude
                    Dim time As DateTime = hour.Time
                    Dim apt As Double = geoLocValue.Value
                Next
            Next
End Using
```

Is the new version missing some functionality? Or am I doing something wrong?

**Comment by 0x1mason** [comment]
No, that should work. That's an error from the underlying native library. Can you share the file that's giving you problems?

**Comment by 0x1mason** [comment]
What sort of grid are you dealing with?,Perhaps it's not supported.

**Comment by DPB61** [comment]
Ah, 2 good questions!
The filename is ds.waveh.bin which isn't much help to anyone so I went back through my notes & couldn't see where the file actually came from.
So I decided to get another one from the NOAA site and did a quick test - the newly download file works!
I believe the file came from here (or another area):
ftp://tgftp.nws.noaa.gov/SL.us008001/ST.expr/DF.gr2/DC.ndfd/AR.alaska/VP.001-003/

Actually I am more interested in global data and I just tested the API with data from here:
http://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/prod/gfs.2017082818/
and it also works perfectly.

My apologies - could have been a corrupted file I guess.
